base_tier_validation/models/tier_validation.py

```diff
@@ -320,6 +327,20 @@ def _check_state_conditions(self, vals):
     def _validate_tier(self, tiers=False):
         self.ensure_one()
         tier_reviews = tiers or self.review_ids
+        waiting_reviews = tier_reviews.filtered(
+            lambda r: r.status == "waiting"
+            or r.approve_sequence_bypass
+            and (self.env.user in r.reviewer_ids)
+        )
```

Comment:
Are parentheses valid?
```suggestion
        waiting_reviews = tier_reviews.filtered(
            lambda r: r.status == "waiting"
            or r.approve_sequence_bypass
            and self.env.user in r.reviewer_ids
        )
```